website/docs/r/google_kms_crypto_key.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "google"
+page_title: "Google: google_kms_key_ring"
+sidebar_current: "docs-google-kms-crypto-key"
+description: |-
+ Allows creation of a Google Cloud Platform KMS CryptoKey.
+---
+
+# google\_kms\_crypto\_key
+
+Allows creation of a Google Cloud Platform KMS CryptoKey. For more information see
+[the official documentation](https://cloud.google.com/kms/docs/object-hierarchy#cryptokey)
+and
+[API](https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.keyRings.cryptoKeys).
+
+A CryptoKey is an interface to key material which can be used to encrypt and decrypt data. A CryptoKey belongs to a
+Google Cloud KMS KeyRing.
+
+~> Note: CryptoKeys cannot be deleted from Google Cloud Platform. Destroying a Terraform-managed CryptoKey will remove it
+from state and delete all CryptoKeyVersions, rendering the key unusable, but **will not delete the resource on the server**.
+
+## Example Usage
+
+```hcl
+resource "google_kms_crypto_key" "my_crypto_key" {
+  name            = "my-crypto-key"
+  key_ring        = "my_project/us-central1/my-key-ring"
+  rotation_period = "100000s"
+}
```

Comment:
It might be nice to include a `google_kms_key_ring` resource here as well, so you could demonstrate using string interpolation for the `key_ring` field.